addons/website_livechat/models/im_livechat_channel.py

```diff
@@ -43,7 +43,16 @@ def _get_livechat_discuss_channel_vals(self, /, *, chatbot_script=None, agent=No
     def create(self, vals_list):
         channels = super().create(vals_list)
         if self.env.context.get("create_from_website"):
+            website = self.env['website'].get_current_website()
+            welcome_bot = self.env['chatbot.script'].search([('title', '=', 'Welcome Bot')], limit=1)
             for channel in channels:
+                website.channel_id = channel.id
```

Comment:
Why changing channel_id inside loop? if there are 5 channels then first iteration will write the channel_id and then second iteration will replace it with second channel id, what is the meaning of this code?